build.rs

```diff
@@ -175,8 +173,6 @@ fn main() {
                     println!("cargo:rustc-link-lib=c");
                     build.file("./deps/wasi_to_unknown.cpp");
                 }
-            } else if !(compile_target_os == "windows" && compile_target_env == "msvc") {
```

Comment:
Is it possible to add a GitHub workflow to ensure your case never gets broken?

Reply:
You mean adding an ubutu+gcc variant to the matrix?

Reply:
Yes

Reply:
> You mean adding an ubuntu+gcc variant to the matrix?

I could do that (i.e. by passing `CXX=g++` in the env), but I don't see an easy way to test that the build _actually_ ends up using gcc...

Reply:
@steveklabnik wdyt?